x-pack/filebeat/input/awscloudwatch/config.go

```diff
@@ -13,20 +13,21 @@ import (
 )
 
 type config struct {
-	harvester.ForwarderConfig `config:",inline"`
-	LogGroupARN               string              `config:"log_group_arn"`
-	LogGroupName              string              `config:"log_group_name"`
-	LogGroupNamePrefix        string              `config:"log_group_name_prefix"`
-	RegionName                string              `config:"region_name"`
-	LogStreams                []*string           `config:"log_streams"`
-	LogStreamPrefix           string              `config:"log_stream_prefix"`
-	StartPosition             string              `config:"start_position" default:"beginning"`
-	ScanFrequency             time.Duration       `config:"scan_frequency" validate:"min=0,nonzero"`
-	APITimeout                time.Duration       `config:"api_timeout" validate:"min=0,nonzero"`
-	APISleep                  time.Duration       `config:"api_sleep" validate:"min=0,nonzero"`
-	Latency                   time.Duration       `config:"latency"`
-	NumberOfWorkers           int                 `config:"number_of_workers"`
-	AWSConfig                 awscommon.ConfigAWS `config:",inline"`
+	harvester.ForwarderConfig          `config:",inline"`
+	LogGroupARN                        string              `config:"log_group_arn"`
+	LogGroupName                       string              `config:"log_group_name"`
+	LogGroupNamePrefix                 string              `config:"log_group_name_prefix"`
+	IncludeLinkedAccountsForPrefixMode *bool               `config:"include_linked_accounts_for_prefix_mode"`
```

Comment:
```suggestion
	IncludeLinkedAccountsForPrefixMode bool               `config:"include_linked_accounts_for_prefix_mode"`
```

Reply:
In order to deal with the nil situation, I think we can add `IncludeLinkedAccountsForPrefixMode: false` under `defaultConfig` function.

Reply:
I switched backed to non-pointer usage - https://github.com/elastic/beats/pull/41206/commits/7d37eb8d57192f4f875d8a432657e8f77c3d0f16

Default (if property not present is false)